packages/kbn-reporting/export_types/csv/types.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { SerializedSearchSourceFields } from '@kbn/data-plugin/common';
```

Comment:
It seems like these types should be in `packages/kbn-reporting/export_types_helpers_server/types.ts`